src/schema/v2/artworks.ts

```diff
@@ -16,10 +16,11 @@ const Artworks: GraphQLFieldConfig<void, ResolverContext> = {
       type: new GraphQLList(GraphQLString),
     },
   }),
-  resolve: (_root, options, { artworksLoader }) => {
+  // @ts-ignore
```

Comment:
This works but I'm not sure how to add this additional key to the type properly or if there's a better way to pass this through. Should I add this argument to the root field instead so it can be passed in as args?

I was able to define an interface to resolve the errors in this file, but ran into other errors in the `v2/schema.ts` file.

```typescript
interface ArtworksResolverContext extends ResolverContext {
  respectParamsOrder: Boolean
}
```

```
src/schema/v2/schema.ts:167:3 - error TS2322: Type '{ artworkAttributionClasses: GraphQLFieldConfig<void, ResolverContext, { [argName: string]: any; }>; article: GraphQLFieldConfig<void, ResolverContext, { ...; }>; ... 36 more ...; usersConnection: GraphQLFieldConfig<...>; }' is not assignable to type 'Thunk<GraphQLFieldConfigMap<any, ResolverContext, { [key: string]: any; }>>'.
  Type '{ artworkAttributionClasses: GraphQLFieldConfig<void, ResolverContext, { [argName: string]: any; }>; article: GraphQLFieldConfig<void, ResolverContext, { ...; }>; ... 36 more ...; usersConnection: GraphQLFieldConfig<...>; }' is not assignable to type 'GraphQLFieldConfigMap<any, ResolverContext, { [key: string]: any; }>'.
    Property 'artworks' is incompatible with index signature.
      Type 'GraphQLFieldConfig<void, ArtworksResolverContext, { [argName: string]: any; }>' is not assignable to type 'GraphQLFieldConfig<any, ResolverContext, { [key: string]: any; }>'.
        Property 'respectParamsOrder' is missing in type 'ResolverContextValues & AvailableLoaders' but required in type 'ArtworksResolverContext'.

167   fields: rootFields,
```

Reply:
Moved `respectParamsOrder` over to a field argument which I think feels a bit better than using the context.